packages/picasso-rich-text-editor/src/LexicalEditor/utils/hasChildDOMNodeTag.test.ts

```diff
@@ -0,0 +1,56 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { JSDOM } from 'jsdom'
+
+import hasChildDOMNodeTag from './hasChildDOMNodeTag'
+
+describe('hasChildDOMNodeTag', () => {
+  // Set up a JSDOM document and window objects
+  const { document } = new JSDOM('').window
+
+  // Test to check if function works with an empty node
+  describe('when node has no children', () => {
+    it('should return false', () => {
```

Comment:
```suggestion
    it('returns false', () => {
```

Due to https://toptal-core.atlassian.net/wiki/spaces/ENG/pages/1120830304/Frontend+Testing+-+Unit+testing#It